popUp.css

```diff
@@ -0,0 +1,192 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+.project-detail-card {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  overflow-x: hidden;
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  overflow-y: scroll;
+  margin: 0 auto;
+  height: 100%;
+  width: 80%;
+}
+
+.modal-prof {
+  width: 100%;
+  background-color: rgb(251, 245, 248);
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.modal-prof .project-liste {
+  padding: 0;
+  list-style-type: circle;
+}
+
+.lister {
+  display: inline-block;
+  list-style-type: circle;
+  margin-right: 20px;
+}
+
+#close-popUp {
+  position: absolute;
+  right: 20px;
+  top: 10px;
+  padding: 0 5px;
+  font-size: 35px;
+  cursor: pointer;
+  color: #1a2236;
+}
+
+.modal-image {
+  width: 100%;
+  height: 100%;
+}
+
+.modal-image img {
+  width: 100%;
+  height: 100%;
+}
+
+.modal-head {
+  position: relative;
+  display: flex;
+  width: 102%;
+  background-color: white;
+}
+
+.modal-head h2 {
+  width: 50%;
+}
+
+.modal-main {
+  width: 95%;
+  display: flex;
+  color: #344563;
+  margin: 0 auto;
+}
+
+.modal-description {
+  width: 70%;
+  font-family: "Poppins", sans-serif;
+  text-align: left;
+  color: #344563;
+  font-size: 18px;
+}
+
+.modal-tech {
+  padding-right: 10px;
+}
+
+.project-lists {
+  display: flex;
+  gap: 8px;
+}
+
+.project-listser {
+  display: flex;
+}
+
+.lisp {
+  list-style: none;
+  background: #ebebff;
+  border-radius: 4px;
+  padding: 5px 10px;
+  margin: 0 2px;
+  font-size: 14px;
+  color: #6070ff;
+}
+
+.list {
+  list-style: none;
+  background: #ebebff;
+  border-radius: 5px;
+  padding: 5px 10px;
+  margin: 0 3px;
+  font-size: 14px;
+  color: #6070ff;
+}
+
+.click-btn {
+  font-size: 15px;
+  color: #396df2;
+  padding: 8px;
+  background-color: transparent;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+  width: 40%;
+  margin: 10px 10px;
+  text-decoration: none;
+  text-align: left;
+  position: relative;
+}
+
+.click-btn img {
+  position: absolute;
+  top: 5px;
+}
+
+.click-btn a {
+  margin-right: 7px;
+  padding-right: 13px;
+  text-decoration: none;
+}
+
+@media only screen and (max-width: 768px) {
+  .modal-main {
+    width: 95%;
+    flex-direction: column;
+    display: flex;
+    color: #344563;
+    margin: 0 auto;
+  }
+
+  .project-listser {
+    display: none;
+  }
+
+  .modal-description {
+    width: 100%;
+    font-family: "Poppins", sans-serif;
+    text-align: left;
+    color: #344563;
+    font-size: 18px;
+  }
+
+  .click-btn {
```

Comment:
- [ ] Kindly fix the size of the buttons and position them a little more to the right to match with the Figma design

![image](https://user-images.githubusercontent.com/93630700/211030947-04987b3e-228f-489f-af6a-11075a33c300.png)